diff --git a/src/main/java/ru/SmrItems/items/ItemCase.java b/src/main/java/ru/SmrItems/items/ItemCase.java
index 0000000..0000000 100644
--- a/src/main/java/ru/SmrItems/items/ItemCase.java
+++ b/src/main/java/ru/SmrItems/items/ItemCase.java
@@ -17,6 +17,7 @@
 public class ItemCase extends Item {
     private final String caseType;
     private static Map<String, List<CaseItem>> allCases;
+    private static boolean hasAttemptedLoad = false;
     private final Random random = new Random();

     public ItemCase(String name, String texture, int maxStackSize, String caseType) {